mussidae/scripts/convert_false_positives_to_ranges.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import absolute_import
+import os
+import numpy as np
+from numpy.lib.recfunctions import append_fields
+import dateutil.parser
+import logging
+import csv
+from cStringIO import StringIO
+import mussidae.time_range_tools as trtools
+from mussidae.time_range_tools import false_positives
+
+
+def download_from_bq(ranges):
+    range_map = {}
+    queries = []
+    for i, rng in enumerate(ranges):
+        (mmsi, start, end, _) = rng
+        gcs_path = base_path + "range_{}.csv".format(i)
+        table = "scratch_{0}".format(i)
+        temp_dest = {'dataset': 'scratch_fishing_score', 'table': table}
+        query = create_query(mmsi, start, end)
+        range_map[gcs_path] = rng
+        queries.append(
+            dict(
+                proj_id=proj_id,
+                query=query,
+                path=gcs_path,
+                temp_dest=temp_dest,
+                compression="NONE"))
+        # if i > 2:
+        #     break # XXX
+    bigq = bqtools.BigQuery()
+    rows = []
+    header = None
+    for gcs_path in bigq.parallel_query_and_extract(queries):
```

Comment:
Any chance you could write a brief comment explaining what this does? In particular I'm not sure what 'rng' is up to... :-)